playkitvr/src/main/java/com/kaltura/playkitvr/DefaultVRPlayerWrapper.java

```diff
@@ -55,10 +61,27 @@ private MDVRLibrary createVRLibrary() {
                     }
                 })
                 .interactiveMode(MDVRLibrary.INTERACTIVE_MODE_TOUCH)
-                .barrelDistortionConfig(new BarrelDistortionConfig().setDefaultEnabled(false).setScale(0.95f))
+                .barrelDistortionConfig(barrelDistortionConfig)
                 .build(((VRView) playerEngine.getView()).getGlSurface());
     }
 
+    @Nonnull
+    private BarrelDistortionConfig extractBarrelDistortionConfig(@Nullable VRSettings vrSettings) {
+        if (vrSettings != null) {
+            VRBarrelDistortionConfig vrBarrelDistortionConfig = vrSettings.getVrBarrelDistortionConfig();
+            if (vrBarrelDistortionConfig != null) {
+                return new BarrelDistortionConfig()
+                        .setParamA(vrBarrelDistortionConfig.getParamA())
```

Comment:
@GouravSna  use names we can understand internally?
vrBarrelDistortionConfig.getParamA

Reply:
Haven't renamed the variable names but given the definition in `VRBarrelDistortionConfig`